docs/isa-declarations/Basics.md

```diff
@@ -0,0 +1,27 @@
+## ISA Declarations
+
+An ISA declaration is a statement of confirmation that needs to be agreed to by a customer before a subscription can be added to an ISA or JISA.
+
+## ISAs
+
+The declaration for an ISA confirms that the customer:
+
+- is over the age of 18
+- has not and will not subscribe more than the permitted allowance in any combination of ISAs in the same tax year
+- has not and will not subscribe to another stocks and share ISA in the same tax year
+- is a resident of the UK (or permitted non-resident)
+
+After completing a declaration, the customer can continue to make subscriptions for the remainder of the tax year. As long as a customer makes at least one subscription in the tax year, the declaration will rollover to the following tax year. This will continue indefinitely until there is a year with no subscriptions.
+
+Following a tax year with no subscriptions, if a customer wishes to further subscribe to their ISA, a new declaration will need to be completed. 
```

Comment:
This isn't true for the first declaration they sign, is it? I think it's valid for two years (assuming 0 contributions)

As a side note, we need to revisit this guide for the new fiscal year, as this rule is being removed I believe?

Reply:
That is true, although I think it complicates understanding. As you say, rules are expected to change in april, so will leave as it is for now